beesdoo_product/models/beesdoo_product.py

```diff
@@ -11,6 +11,19 @@
 _logger = logging.getLogger(__name__)
 
 
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    profit_margin = fields.Float(string="Product Margin [%]")
+
+    @api.multi
+    @api.constrains("profit_margin")
+    def _check_margin(self):
+        for product in self:
+            if product.profit_margin < 0.0:
+                raise UserError(_("Percentages for Profit Margin must > 0."))
```

Comment:
From my understanding error message should be "Percentages for Profit Margin must >= 0."

Reply:
That's right! I'll fix it.

Reply:
It was a copy pase from line 324. I'll fix it there as well.